lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -267,19 +267,26 @@ def get_course_assignments(self, model: CourseEnrollment) -> Optional[Dict[str,
             self.context.get('request'),
             include_past_dates=True
         )
-        next_assignment = None
+        next_assignment_all = []
+        next_assignment = []
         past_assignment = []
```

Comment:
```suggestion
        future_assignments = []
        past_assignments = []
```
Change names a bit: plural for both and `future_assignments`

Reply:
`next_assignments` will be initialized later